modules/services/docassemble/vars.tf

```diff
@@ -0,0 +1,69 @@
+variable "environment" {
+  description = "Environment name"
+}
+
+variable "container_image" {
+  description = "Docker image name"
+  default     = "jhpyle/docassemble:latest"
+}
+
+variable "container_memory_reservation" {
+  description = "Memory reservation for containers"
+  default     = 256
+}
+
+variable "vpc_id" {
+  description = "VPC Id to be used by the LB"
+}
+
+variable "lb_listener_id" {
+  description = "LB listener id"
+}
+
+variable "ecs_cluster_id" {
+  description = "ECS cluster id where the app will run"
+}
+
+variable "domain" {
+  description = "FQDN where app will be available"
+}
+
+variable "desired_count" {
+  description = "Number of instances to be run"
+  default     = 1
+}
+
+variable "log_retention_in_days" {
+  description = "Cloudwatch logs retention"
+  default     = 3
+}
+
+// Docassemble app variables
+variable "db_backups" {
+  description = "Do db backups"
+}
+
+variable "db_host" {
+  description = "Database host"
+}
+
+variable "db_name" {
+  description = "Database name"
+}
+
+variable "db_password" {
+  description = "Database password"
+}
+
+variable "db_user" {
+  description = "Database user"
+}
+
+variable "s3_bucket" {
+  description = "AWS S3 bucket name"
```

Comment:
This types of descriptions are ok? Or it is because the PR is half way?

Reply:
I think is pretty self explanatory. How would you improve it?

Reply:
They are self explanatory and very obvious, that is the why I feel they were wrong... I though this was like: 

_"Bucket to store images from community"_, like description with the propose of the item you're defining. Otherwise, I don't see the point to have `variable s3_bucket` and have a description _(because all s3_buckets will have the same description)_

Reply:
**Disclaimer:** they may be perfectly fine, I just make sure you challenge the way you do things to maybe discover they can be better. 👌

Reply:
I guess is a good practice to always define the description for variables, even when the variable name is pretty self-explanatory. I don't want to overthink these description for now but feel free to open a new PR to make them better.